flexget/plugins/input/my_anime_list.py

```diff
@@ -58,67 +60,43 @@ class MyAnimeList(object):
     @cached('my_anime_list', persist='2 hours')
     def on_task_input(self, task, config):
         entries = []
-        parameters = {'u': config['username'], 'status': 'all', 'type': 'anime'}
-        selected_status = config.get('status', list(STATUS.values()))
-        selected_types = config.get('type', list(ANIME_TYPE.values()))
+        selected_status = config.get('status', list(STATUS.keys()))
+        selected_types = config.get('type', list(ANIME_TYPE))
 
         if not isinstance(selected_status, list):
             selected_status = [selected_status]
 
         if not isinstance(selected_types, list):
             selected_types = [selected_types]
 
+        for i in range(len(selected_status)):
```

Comment:
Looks like you can use enumerate here

Reply:
Fixed that and the other comments below. Should open a new PR or can a commit be added to this one?